.github/workflows/build-and-upload-release.yml

```diff
@@ -0,0 +1,122 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Build And Upload Artifact
+env:
+  # Use terraform assume role for uploading to ecr
+  AWS_ASSUME_ROLE: ${{ secrets.TERRAFORM_AWS_ASSUME_ROLE }}
+  ECR_OPERATOR_STAGING_IMAGE: ${{ secrets.ECR_OPERATOR_STAGING_IMAGE }}
+  ECR_OPERATOR_RELEASE_IMAGE: ${{ secrets.ECR_OPERATOR_RELEASE_IMAGE }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        description: 'Release Artifact'
+        required: false
+        default: false
+        type: boolean
+      staging_postfix:
+        description: 'Staging Artifact Postfix'
+        required: false
+        default: '-operator-staging-dispatch'
+        type: string
+  workflow_call:
+    inputs:
+      release:
+        description: 'Release Artifact'
+        required: false
+        default: false
+        type: boolean
+      staging_postfix:
```

Comment:
This might not work with our e2e tests since our e2e tests are just looking at our secret `ECR_OPERATOR_STAGING_IMAGE` which includes the repo and tag. 

I like the idea of postfix though -- maybe we can change this to be the tag?

So we can create a new secret `ECR_OPERATOR_STAGING_IMAGE_REPO` which will just be the repo, and we add the tag to it later to be:

```
tags: ${{ env.ECR_OPERATOR_STAGING_IMAGE_REPO }}:${{ inputs.staging_tag }}
```

and we can set the tag to be `:staging` for our release workflow and `:integ-test` on our push to main

Reply:
Changed to using tags instead of postfixes and now have the e2e test use the tag instead of latest